core/src/test/scala/unit/kafka/tools/MirrorMakerTest.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.kafka.tools
+
+import kafka.consumer.BaseConsumerRecord
+import kafka.tools.MirrorMaker
+import org.apache.kafka.common.record.TimestampType
+import org.junit.Assert._
+import org.junit.Test
+
+
+class MirrorMakerTest {
+
+  @Test
+  def testDefaultMirrorMakerMessageHandler() {
+    val now = System.currentTimeMillis()
```

Comment:
It would be better to use a fixed timestamp instead of `System.currentTimeMillis`. The reason is that the latter could pass in a scenario where a new timestamp is created via `System.currentTimeMillis`, but the time between both invocations is really short.